proto/recon.proto

```diff
@@ -145,3 +145,15 @@ message RecognizePlacesResponse {
   }
   repeated Item items = 1;
 }
+
+// RecogPlace contains info of a place for RecognizePlaces method.
+message RecogPlace {
+  repeated string words = 1;
```

Comment:
These parts of the name right?  We may soon want to add multiple names.  So either make it `repeated string names = 1;` that we split,
or 

```
message NameParts {
  repeated string words = 1;
}

repeated NameParts names = 1;
```

Reply:
done